lib/data/context.js

```diff
@@ -46,6 +48,18 @@ module.exports = function (context) {
   };
 };
 
+function replaceTitle(title, dataRow) {
+  if (typeof dataRow.data === 'object') {
+    return `${title} | ${JSON.stringify(dataRow.data)}`;
+  }
+
+  if (Array.isArray(dataRow.data)) {
+    return `${title} | {${JSON.stringify(dataRow.data)}}`;
```

Comment:
`{${JSON.stringify(dataRow.data)}}` - additional `{}` is not needed here

Reply:
additional `{}` needed, because in file `ui.js` on 139 line you used regexp `/( \| {.+})?$/g` where used `{}` and when data is array of string codeceptjs run only last data